autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -117,9 +166,15 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Use `auto_stack` to maximize predictive accuracy; use `hyperparameter_tune` if you prefer to deploy just a single model rather than an ensemble.
         feature_prune : bool, default = False
             Whether or not to perform feature selection.
-        refit_full : bool, default = False
+        refit_full : bool or str, default = False
             Whether to retrain all models on all of the data (training + validation) after the normal training procedure.
-            This is equivalent to calling `predictor.refit_full()` after training.
+            This is equivalent to calling `predictor.refit_full(model=refit_full)` after training.
+            If `refit_full=True`, it will be treated as `refit_full='all'`.
+            If `refit_full=False`, refitting will not occur.
+            Valid str values:
+                `all`: refits all models.
+                `best`: refits only the best model (and its ancestors).
```

Comment:
(and its ancestors if it is a stacker model)

Reply:
Added